README.md

```diff
@@ -172,3 +172,24 @@ If you build the app on your own using Android Studio `(Build -> Build APK)` and
 
 This problem occurs building other apps as well.
 
+#### gradlew Failure: `FAILURE: Build failed with an exception.`
+
+If you encounter an error similar to this when running `gradlew`:
```

Comment:
To make this a wee bit more generic, maybe we just say if you encounter an error. Try googling for the error. If that doesn't work, try setting JAVA_HOME.